scripts/functions/installer

```diff
@@ -1100,8 +1100,15 @@ setup_user_profile_rc()
     do
       touch "$profile_file"
       printf "%b" "
-# Add RVM to PATH for scripting. Make sure this is the last PATH variable change.
+###############################################################################
+# At $(date --iso-8601=s), the RVM installer added this block (and also created
+# this file if it did not already exist).
+#########
+#
+# Add RVM to PATH for scripting. Ensure this is the last PATH variable change.
```

Comment:
This line could be written in a more clear way, don't you think? Maybe "Include RVM in the PATH. Ensure it is always at the end of the PATH".